src/plugins/drawing/components/drawing-layer.tsx

```diff
@@ -310,10 +323,76 @@ export class DrawingLayerView extends React.Component<DrawingLayerViewProps, Dra
                 strokeWidth="1.5"
                 strokeDasharray="10 5"
                 pointerEvents={"none"}
-               />;
+               />
+               {resizers}
+             </g>;
     });
   }
 
+  public renderResizeHandle(object: DrawingObjectType, corner: string, x: number, y: number, color: string) {
+    const resizeBoxOffset = SELECTION_BOX_RESIZE_HANDLE_SIZE/2;
+
+    return <rect key={corner} data-corner={corner} className={"resize-handle " + corner}
+                x={x-resizeBoxOffset} y={y-resizeBoxOffset} 
+                width={SELECTION_BOX_RESIZE_HANDLE_SIZE} height={SELECTION_BOX_RESIZE_HANDLE_SIZE}
+                stroke={color} strokeWidth="1" fill="#FFF" fillOpacity="1"
+                onMouseDown={(e) => this.handleResizeStart(e, object)}
+          />;
+  }
+
+  private handleResizeStart(e: React.MouseEvent<SVGRectElement, MouseEvent>, object: DrawingObjectType) {
+    e.stopPropagation();
+    e.preventDefault();
+    const handle = e.currentTarget;
+    handle.classList.add('active');
+    const corner = handle.dataset.corner;
+    
+    const start = this.getWorkspacePoint(e);
+    const origBoundingBox = object.boundingBox;
+    // The original size of the element is used to avoid making it 0 or negative size.
+    const origWidth = origBoundingBox.se.x - origBoundingBox.nw.x;
+    const origHeight = origBoundingBox.se.y - origBoundingBox.nw.y;
+
+    const handleResizeMove = debounce((e2: MouseEvent) => {
+      e2.stopPropagation();
+      e2.preventDefault();
+      // Check if mouse is within the drawtool canvas; if not do nothing.
+      if (!((this.svgRef as unknown) as Element).matches(':hover')) {
+        return;
+      }
+      const current = this.getWorkspacePoint(e2);
+      if (!start || !current || !corner) return;
+      const dx = current.x - start.x;
+      const dy = current.y - start.y;
+      // Determine which edges the user wants to move, and how much
+      // Constrain these to make sure the image won't get 0 or negative size.
+      const deltas = {
+        top:    corner.charAt(0)==='n' ? Math.min(dy,   origHeight-1)  : 0,
+        right:  corner.charAt(1)==='e' ? Math.max(dx, -(origWidth -1)) : 0,
+        bottom: corner.charAt(0)==='s' ? Math.max(dy, -(origHeight-1)) : 0,
+        left:   corner.charAt(1)==='w' ? Math.min(dx,   origWidth -1)  : 0
+      };
+
+      object.dragBounds(deltas);
+      
+    }, 10);
+  
+    const handleResizecomplete = (e2: MouseEvent) => {
+      e2.stopPropagation();
+      e2.preventDefault();
+      window.removeEventListener("mousemove", handleResizeMove);
+      window.removeEventListener("mouseup", handleResizecomplete);
+      handle.classList.remove('active');
+      object.adoptDragBounds();
+    };
+
+    window.addEventListener("mousemove", handleResizeMove);
+    window.addEventListener("mouseup", handleResizecomplete);
+  }
+
+
+  //we want to populate our objectsBeingDragged state array
+
```

Comment:
Is this comment still relevant?

Reply:
I think this one was removed in my other PR